src/backend/metal/src/command.rs

```diff
@@ -1653,6 +1653,8 @@ impl com::RawCommandBuffer<Backend> for CommandBuffer {
         T: IntoIterator,
         T::Item: Borrow<SubresourceRange>,
     {
+        let _ap = AutoreleasePool::new();
```

Comment:
Does this fix the leak?

Reply:
possible! my motivation was to avoid 2 autorelease pools during a regular `begin_render_pass`, so now I removed the inner one and made sure all the callers set up one.